route-rs-runtime/src/element/identity_elements.rs

```diff
@@ -17,7 +17,10 @@ impl<A> IdentityElement<A> {
     }
 }
 
-impl<A> Element for IdentityElement<A> {
+impl<A> Element for IdentityElement<A>
```

Comment:
Can this not have the type constraint `A: Send`?

Reply:
That seems reasonable, will change

Reply:
done